app/konnect/getting-started/index.md

```diff
@@ -2,60 +2,21 @@
 title: Get started with Konnect
 ---
 
-## Quickstart guide
-
-New to {{site.konnect_saas}}? Get started with the basics through the web app:
-
-1. **Create a {{site.konnect_short_name}} account**:
-
-    [Create an account &gt;](/konnect/getting-started/access-account/)
-
-1.  **Set up a data plane node**:
-
-    Set up your first data plane node for proxying traffic.
-
-    Start with a data plane proxy so that when your service configuration is
-    ready, your services are immediately connected, configured,
-    and available for testing.
-
-    [Set up a data plane node &gt;](/konnect/getting-started/configure-data-plane-node/)
-
-
-2.  **Create a Gateway service**:
-
-    Proxy a service using the {{site.konnect_short_name}} Gateway Manager
-
-    After you've configured a service and the data plane proxy is ready, testing
-    is the next step. Here you set up a route to a mocking API service,
-    then proxy traffic through the route.
-
-
-    [Create a service &gt;](/konnect/getting-started/configure-service/)
-
-3.  **Productize an API**:
-    
-    Using API Products, you can bundle and manage every service in your
-    entire architecture. In this step, you create the first version of a service,
-    adding the first entry to your catalog.
-    
-
-    [Create an API product version &gt;](/konnect/getting-started/productize-service/)
-
-## Share APIs and enable development
-
-Ready to share your API with developers?
-Use API Products to manage your API product documentation, then publish it to the Dev Portal:
-
-1. **Upload your spec to {{site.konnect_short_name}} and publish to Dev Portal**
-
-    Upload a markdown file to describe your service, and add an OpenAPI spec for any version of the API product.
-    Then, publish your API product documentation to the Dev Portal and preview it from the developer perspective.
-
-    [Import docs and publish an API product &gt;](/konnect/getting-started/publish-service/)
-
-1. **Register an application**
-
-    Expose your service to developers and register an application
-    against the service.
-
-    [Expose service and register an application &gt;](/konnect/getting-started/app-registration/)
+<div class="docs-grid-install max-3">
```

Comment:
I'm struggling with this page but only because there are limitations in our docs site not because of what you have. 

Here are some suggestions: 
1. We could replace the tiles in this page https://docs.konghq.com/konnect/ with your tiles removing the existing 'get started', and 'set up data plane nodes'. 




2. We could use mermaid with links to make it be more center on the existing page and maybe look nicer 
![image](https://github.com/Kong/docs.konghq.com/assets/23319190/600113d2-387f-43f2-876d-47f4b8fdcefd)

3. Maybe in a table?

Reply:
I like the first suggestion a lot! I'll go with that